src/node_file.h

```diff
@@ -52,6 +52,8 @@ enum class FsStatFsOffset {
   kFsStatFsFieldsNumber
 };
 
+enum class ModeOpType { access = 0, copyFile };
```

Comment:
Style. Either:
```suggestion
enum class ModeOpType { ACCESS, COPYFILE };
```
Or:
```suggestion
enum class ModeOpType { kAccess, kCopyFile };
```
PS: you don't need to assign a value so I left that out.

Reply:
Ops, I added this line by a mistake. I'm using the UV_FS_* flags.